recipes/mne-microstates/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "mne-microstates" %}
+{% set version = "0.3" %}
+{% set pypi_filename = "mne_microstates" %}
```

Comment:
This is indeed a problem with some URLs but in that case, it's better to just hardcode it.